AdaptiveUiCodelab/gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
2022

Reply:
This is autogenerated and is same for all samples.